### ISSUE TITLE
404 on loaded Livewire js script

### ISSUE THREAD

**Issue by fgschafer** [rainlab/livewire-plugin#2]
I have a 404 error on the livewire script js (so it doesn’t work). In my case OC is installed in a subfolder, and even if I set the link policy to force, its trying to get the file without taking into account the whole path with the subfolder in it.

The script is added like this (without the whole path):
`<script src="/livewire/livewire.js?id=90730a3b0e7144480175" data-turbo-eval="false" data-turbolinks-eval="false" ></script>
`

**Comment by daftspunk** [comment]
Fixed in v1.0.2

**Comment by najbo** [comment]
Hello @daftspunk, 

I have the same problem that occurs now. Error 404 on livewire/livewire.js file. 
I have a {% styles %} and {% scripts %} tag in my layout file. 

This is my html output file : 

```
<script src="https://my.domain.tld/livewire/livewire.js?id=4ce12f49" data-csrf="..." data-update-uri="/livewire/update" data-navigate-once="true"></script>
```

Running ```php artisan plugin:refresh RainLab.LiveWire``` results in this html outpout :

```
<script src="https://my.domain.tld/vendor/livewire/livewire.min.js?id=4ce12f49" data-csrf="..." data-update-uri="/livewire/update" data-navigate-once="true"></script>
```

So it adds a /vendor folder in the script.
My server let me access to the https://my.domain.tld/livewire/livewire.js file. The https://my.domain.tld/vendor/livewire/livewire.js return 404 again.

I also tried to run ```php artisan vendor:publish --force --tag=livewire:assets```. Here is the console output : 

```
INFO  Publishing [livewire:assets] assets.

Copying directory [vendor/livewire/livewire/dist] to [vendor/livewire] ............... DONE
```

Note that I don't use the public folder.
